vector/src/main/java/fr/gouv/tchap/activity/TchapLoginActivity.java

```diff
@@ -672,28 +574,29 @@ public void onBackPressed() {
         switch (mMode) {
             case MODE_ACCOUNT_CREATION:
             case MODE_LOGIN:
-                mMode = MODE_START;
-                onClick();
-                refreshDisplay();
+                Log.d(LOG_TAG, "## fallback to initial screen");
+                fallbackToStartMode();
```

Comment:
Why removing `onClick();`?
I think `fallbackToStartMode()` may contains `onClick()` to factorize (please remove other call to `onClick()` if followed by `fallbackToStartMode()`).